lib/evss/base_service.rb

```diff
@@ -50,7 +50,7 @@ def vaafi_headers
         # Vary by user
         'va_eauth_firstName' => @user.first_name,
         'va_eauth_lastName' => @user.last_name,
-        'va_eauth_issueinstant' => @user.issue_instant,
+        'va_eauth_issueinstant' => @user.login_time.utc.iso8601,
```

Comment:
Can we make the assumption that `login_time` is already in UTC (it is now when you set it in the model https://github.com/department-of-veterans-affairs/vets-api/pull/149/files#diff-4676c008b11a5480d73d4a6de01e45b9R28 and I would prefer any time variable to always be in UTC) then you can avoid calling `.utc` again here